Gordon360/Services/HousingService.cs

```diff
@@ -135,6 +135,31 @@ public AA_ApartmentHalls[] GetAllApartmentHalls()
             return result;
         }
 
+        /// <summary>
+        /// Calls a stored procedure that tries to get the editor ID of a given application ID
+        /// </summary>
+        /// <param name="applicationID"> The application ID for which the editor ID would be </param>
+        /// <returns>
+        /// The id of the editor or
+        /// null if the user is a member but not an editor of a given application
+        /// </returns>
+        public string GetEditorID(int? applicationID)
+        {
+            IEnumerable<ApartmentApplicationViewModel> appResult = null;
+
+            SqlParameter applicationIDParam = new SqlParameter("@APPLICATION_ID", applicationID);
+
+            appResult = RawSqlQuery<ApartmentApplicationViewModel>.query("GET_AA_EDITOR_BY_APPID, @APPLICATION_ID", applicationIDParam); // run stored procedure
```

Comment:
This can simply be a string since it only returns data from a single column of the table
```suggestion
            IEnumerable<string> editorResult= null;

            SqlParameter applicationIDParam = new SqlParameter("@APPLICATION_ID", applicationID);

            editorResult = RawSqlQuery<string>.query("GET_AA_EDITOR_BY_APPID, @APPLICATION_ID", applicationIDParam); // run stored procedure
```